transfer.md

```diff
@@ -59,9 +61,38 @@ There are other equivalents for these tools
 | `docker save`   | [`kpod save`](./docs/kpod-save.1.md)      |
 | `docker stop`   | [`kpod stop`](./docs/kpod-stop.1.md)      |
 | `docker tag`    | [`kpod tag`](./docs/kpod-tag.1.md)        |
+| `docker umount` | [`kpod umount`](./docs/kpod-umount.1.md)  |
 | `docker unpause`| [`kpod unpause`](./docs/kpod-unpause.1.md)|
 | `docker version`| [`kpod version`](./docs/kpod-version.1.md)|
 | `docker wait`   | [`kpod wait`](./docs/kpod-wait.1.md)   |
 
 *** Use `kpod exec` to enter a container and `kpod logs` to view the output of pid 1 of a container.
 **** Use mount to take advantage of the entire linux tool chain rather then just cp.  Read [`here`](./docs/kpod-cp.1.md) for more information.
+
+## Missing commands from kpod
+
+Those Docker commands currently do not have equivalents in `kpod`:
+
+ * `docker container`
+ * `docker image`
+ * `docker network`
+ * `docker node`
+ * `docker plugin`
+ * `docker secret`
+ * `docker service`
+ * `docker stack`
+ * `docker swarm`
+ * `docker system`
+ * `docker volume`
+ 
+## Missing commands from Docker
+
+The following kpod commands do not have a Docker equivalent:
+
+* [`kpod events`](./docs/kpod-events.1.md)
+* [`kpod port`](./docs/kpod-port.1.md)
+* [`kpod rename`](./docs/kpod-rename.1.md)
```

Comment:
I don't understand this section.  It looks like these are additional commands that are in docker CLI but not in kpod?
If you are listing commands that are in kpod but not in docker, then you would add

kpod mount
kpod umount

Reply:
it's the opposite, as the above context says:

> The following kpod commands do not have a Docker equivalent:

i fixed the title to try to remove that ambiguity

Reply:
I see docker events, port rename, ... in docker-1.13,  Did they drop these from later releases?

Reply:
rah. i got those backwards, it seems. :(

Reply:
fixed.